app/models/announcement.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: announcements
+#
+#  id         :bigint           not null, primary key
+#  category   :integer
+#  content    :text             not null
+#  deleted_at :datetime
+#  image      :string
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Announcement < ApplicationRecord
+  # TODO: Uncomment this when appropiatte gem configured.
+  # act_as_paranoid
+
+  validates :name, presence: true, length: { in: 1..254 }
+  validates :content, presence: true, length: { minimum: 1 }
```

Comment:
En principo se podría dejar solo el `presence: true`, porque el ticket no pide validaciones específicas. En todo caso lo agregamos más adelante.

Reply:
Si bien el ticket no solicita validaciones, el campo `name`, esta acotado al máximo correspondiente al tipo de dato que se solicita en el diseño de la DB.
Por otro, el campo `content`, sólo está haciendo cumplir del lado del modelo la condición de NOT NULL, también requerida.

Reply:
Me auto corrijo, no, los demás campos no estan como NO NULL en la db.

Reply:
Ya esta corregido, asi si lo queres mergear mañana ya está. Saludos!